cmd2/cmd2.py

```diff
@@ -467,6 +467,11 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, *,
         # Otherwise it can be set to any custom key to meet your needs.
         self.matches_sort_key = ALPHABETICAL_SORT_KEY
 
+        # The maximum number of CompletionItems to display during tab completion. If the number of completion
+        # suggestions exceeds this number, they will be displayed in the typical columnized format and will
+        # not include the description value of the CompletionItems.
+        self.max_completion_items = 50
```

Comment:
Would it be useful for this to be in `settable`?

Reply:
Done